lib/pages/chat/chat.dart

```diff
@@ -322,6 +327,67 @@ class ChatController extends State<Chat>
     }
   }
 
+  List<Event?> _getUnreadEvents() {
+    final fullyRead = room!.fullyRead;
+    final lastIndexReadEvent = timeline?.events.indexWhere(
+      (event) => fullyRead == event.eventId,
+    );
+    if (lastIndexReadEvent != null && lastIndexReadEvent > 0) {
+      final afterFullyRead = timeline?.events.getRange(0, lastIndexReadEvent);
+      final unreadEvents = afterFullyRead?.map((fullyReadEvent) {
+        if (fullyReadEvent.senderId != client.userID) {
+          return fullyReadEvent;
+        }
```

Comment:
why not use `where` ?
```suggestion
      final unreadEvents = afterFullyRead?.where(fullyReadEvent => fullyReadEvent.senderId != client.userID);
```